string_permutation_problem/permute.py

```diff
@@ -0,0 +1,26 @@
+def permute(s):
+    out = []
+    
+    #Base Case
+    if len(s) == 1:
+        out = [s]
+        
+    else:
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `python spacing`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmp3zw2_ifm/string_permutation_problem/permute.py
+++ b/tmp/tmp3zw2_ifm/string_permutation_problem/permute.py
@@ -4,7 +4,7 @@
     #Base Case
     if len(s) == 1:
         out = [s]
-        
+
     else:
         #For every letter in String
         for i, let in enumerate(s):
```